Makefile

```diff
@@ -36,7 +36,7 @@ prepare-test:
 	sudo systemctl status -l faasd-provider --no-pager
 	sudo systemctl status -l faasd --no-pager
 	curl -sSLf https://cli.openfaas.com | sudo sh
-	sleep 120 && sudo journalctl -u faasd --no-pager
+	echo "Sleeping for 2m" && sleep 120 && sudo journalctl -u faasd --no-pager
```

Comment:
is this 2m sleep what finally made it work?

Reply:
No, it wasn't added, it was always there. A print statement was added for the logs.